apps/prairielearn/src/components/InstructorInfoPanel.html.ts

```diff
@@ -0,0 +1,372 @@
+      : ''}
+    <div class="d-flex flex-wrap">
+      <div class="pr-1">Title:</div>
+      <div>${question.title}</div>
+    </div>
+  `;
+}
+
+function VariantInfo({ variant }: { variant?: Variant & { formatted_date: string } }) {
+  if (variant == null) return '';
+
+  const duration =
+    // Some legacy queries still return the duration as a string, so parse it before formatting
+    typeof variant.duration === 'string'
+      ? IntervalSchema.parse(variant.duration)
+      : variant.duration ?? 0;
+  return html`
+    <div class="d-flex flex-wrap">
+      <div class="pr-1">Started at:</div>
+      <div>${variant.formatted_date}</div>
+    </div>
+    <div class="d-flex flex-wrap">
+      <div class="pr-1">Duration:</div>
+      <div>${formatInterval(duration)}</div>
+    </div>
+    <div class="d-flex flex-wrap pb-2">
+      <div class="pr-1">
+        <button class="btn btn-link" data-toggle="collapse" data-target="#instructorTrue_answer">
+          Show/Hide answer
+        </button>
+      </div>
+      <div class="collapse" id="instructorTrue_answer">
+        <code>${JSON.stringify(variant.true_answer)}</code>
+      </div>
+    </div>
+  `;
+}
+
+function AssessmentInstanceInfo({
+  assessment,
+  assessment_instance,
+}: {
+  assessment?: Assessment;
+  assessment_instance?: AssessmentInstance & { formatted_date: string };
+}) {
+  if (assessment == null || assessment_instance == null) return '';
+
+  const instructorUrlPrefix = `${config.urlPrefix}/course_instance/${assessment.course_instance_id}/instructor`;
+  const duration =
+    // Some legacy queries still return the duration as a string, so parse it before formatting
+    typeof assessment_instance.duration === 'string'
+      ? IntervalSchema.parse(assessment_instance.duration)
+      : assessment_instance.duration ?? 0;
+
+  return html`
+    <hr />
+    <h5 class="card-title">Assessment Instance:</h5>
+    <div class="d-flex flex-wrap">
+      <div class="pr-1">AID:</div>
+      <div>
+        <a href="${instructorUrlPrefix}/assessment/${assessment.id}">${assessment.tid}</a>
+      </div>
+    </div>
+
+    <div class="d-flex flex-wrap">
+      <div class="pr-1">Started at:</div>
+      <div>${assessment_instance.formatted_date}</div>
+    </div>
+
+    <div class="d-flex flex-wrap">
+      <div class="pr-1">Duration:</div>
+      <div>${formatInterval(duration)}</div>
+    </div>
+
+    <div class="pb-2">
+      <a href="${instructorUrlPrefix}/assessment_instance/${assessment_instance.id}">View log</a>
+    </div>
+  `;
+}
+
+function ManualGradingInfo({
+  instance_question,
+  assessment,
+  questionContext,
+}: {
+  instance_question?:
+    | (InstanceQuestion & {
+        assigned_grader_name?: string | null;
+        last_grader_name?: string | null;
+      })
+    | null;
+  assessment?: Assessment | null;
+  questionContext: QuestionContext;
+}) {
+  if (
+    instance_question == null ||
+    assessment == null ||
+    instance_question.status === 'unanswered'
```

Comment:
This is new functionality: if the instance question is unanswered, don't show the manual grading section. Previously, this would be shown with the status "Graded" (since it does not require manual grading), and with a link to "Grade" that would trigger an error (since there is no gradable submission).